### ISSUE TITLE
droidhole fails to load graphs from other server when main server offline

### ISSUE THREAD

**Issue by Jannik44** [JGeek00/droid-hole#51]
Hello,
i have an issue here, i have a local server in my network as 1st server in the list and another one wich is reachable from the outside internet

1. problem is now when i open the droidhole app, it defaults to the 1st one in the list
2. problem is that it cant load the graphs from the reachable server when the 1st one in the list is unreachable, only the numbers
![Screenshot_2024-09-16-17-05-41-027_com jgeek00 droid_hole_1](https://github.com/user-attachments/assets/e6493e25-8538-4d03-b155-fc6bc47d5fde)
![Screenshot_2024-09-16-17-06-27-761_com jgeek00 droid_hole_1](https://github.com/user-attachments/assets/0e719914-2648-4e8d-8058-5db5a3413f6e)

**Comment by JGeek00** [comment]
Hi. Which server do you have selected as default?

**Comment by JGeek00** [comment]
That's the correct behavior. The app will always try to connect to the default server on start. If you want to connect to a different server you have to do it manually.

**Comment by Jannik44** [comment]
yes of course, but it will fail on the other one as well then

**Comment by Jannik44** [comment]
yes exactly, only the diagrams fail, the numbers on the top are working for some reason

**Comment by JGeek00** [comment]
This app is deprecated and won't receive more updates. Check the readme for more information.
